src/main/java/frc/robot/Constants.java

```diff
@@ -1,84 +1,26 @@
 package frc.robot;
 
-import com.ctre.phoenix6.signals.InvertedValue;
-import com.ctre.phoenix6.signals.NeutralModeValue;
-import com.ctre.phoenix6.signals.SensorDirectionValue;
-
-import edu.wpi.first.math.geometry.Rotation2d;
-import edu.wpi.first.math.geometry.Translation2d;
-import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
 import edu.wpi.first.math.util.Units;
-
-import frc.lib.ModuleConstants;
+import frc.robot.generated.TunerConstants;
 
 public final class Constants {
     public static final class ControllerConstants {
         public static final int kControllerID = 0;
         public static final double kControllerDeadband = 0.15;
     }
 
-    //Units of distance should be in metric
     public static final class DrivetrainConstants {
-        public static final double kWheelbase = Units.inchesToMeters(27); //Guessed
-        public static final double kTrackWidth = Units.inchesToMeters(27);
-
-        public static final double kWheelDiameter = Units.inchesToMeters(4);
-        public static final double kWheelCircumference = kWheelDiameter * Math.PI;
-
-        public static final SwerveDriveKinematics kKinematics = new SwerveDriveKinematics( //Module locations are clockwise starting from the front left module
-            new Translation2d(kWheelbase/2, kTrackWidth/2),
-            new Translation2d(kWheelbase/2, -kTrackWidth/2),
-            new Translation2d(-kWheelbase/2, -kTrackWidth/2),
-            new Translation2d(-kWheelbase/2, kTrackWidth/2)
-        );
-
-        //TODO ensure IDs match robot
-        public static final ModuleConstants kFrontLeftModule = new ModuleConstants(1, 5, 9, Rotation2d.fromDegrees(28.82), "Front Left Module");
-        public static final ModuleConstants kFrontRightModule = new ModuleConstants(2, 6, 10, Rotation2d.fromDegrees(132.09), "Front Right Module");
-        public static final ModuleConstants kBackRightModule = new ModuleConstants(3, 7, 11, Rotation2d.fromDegrees(196.00), "Back Right Module");
-        public static final ModuleConstants kBackLeftModule = new ModuleConstants(4, 8, 12, Rotation2d.fromDegrees(237.30), "Back Left Module");
-
-        public static final double kDriveGearing = 6.75; //6.75 : 1; MK4 L2 ratio
-        public static final double kAzimuthGearing = 12.8; //12.8 : 1; steering ratio for all SDS modules
+        // Most drivetrain-specific constants have been moved to TunerConstants.java, which is
+        // maintained by Phoenix Tuner. If any drivetrain constants need to be modified, or new constants added, do so here;
+        // TunerConstants.java may be overwritten in the future
```

Comment:
I'd rather not separate our Constants if possible; bit redundant to have a `Constants.java` that doesn't contain all of the constants -- or, at least, all of the public constants that'll be accessed by the rest of the code.